apluslms_roman/builder.py

```diff
@@ -60,11 +76,13 @@ def __init__(self, backend_class=None, settings=None):
 
         name = getattr(backend_class, 'name', None) or backend_class.__name__.lower()
         env_prefix = name.upper() + '_'
-        env = {k: v for k, v in environ.items() if k.startswith(env_prefix)}
+        env = load_from_env(env_prefix, '.')
+        print("env without setting:", env)
         if settings:
             for k, v in settings.get(name, {}).items():
                 if v is not None and v != '':
-                    env[env_prefix + k.replace('-', '_').upper()] = v
+                    env[k] = dict(v) if isinstance(v, ChangesDict) else v
+        print("env after setting:", env)
```

Comment:
Thre shouldn't be any `print` statements in other files than `cli.py` in the library. You can find all places of print in the roman with `git grep -F print '*.py'` for reference.

Reply:
Replace with logging.debug()